disco_aws_automation/asiaq_cli.py

```diff
@@ -100,7 +106,7 @@ def _update_s3_configs(self, sandbox_name):
         s3_bucket = boto3.resource("s3").Bucket(name=bucket_name)
         for sync_line in config_sync_option.split("\n"):
             local_name, remote_dir = sync_line.split()
-            local_config_path = normalize_path("sandboxes", sandbox_name, local_name)
+            local_config_path = normalize_path("sandboxes", config_dir, local_name)
```

Comment:
Does this mean that `config_dir` is a required arg to the command?

Reply:
It is optional.

Reply:
Oh I see